### PR TITLE
DEXA-316 - docs conversion script

### PR DESCRIPTION
**Pull request by arkadiuszkrykM**
This changed:
- we have altered our auto-generated doc filenames from `./_internal_.Conference.md` to `./internal.Conference.md` - mainly because underscores were causing weird md format issues
- we have improved typedoc configuration a bit and removed unnecessary things (like breadcumbs)
- added convert script

### CODE REVIEW

File: scripts/convertDocs.js [hunk 1]
```diff
@@ -0,0 +1,180 @@
...
+
+  FILE_PATHS.forEach(convertDoc);
+
+  console.log('Conversion successful!');
+})();
+
+function createDocHeader(originalFilename, slug, order) {
+  return `
+---
+apiVersion: 1.0
```
Comment (ppolus-miquido) [inline scripts/convertDocs.js:54]: Could we take it from package.json?

File: scripts/convertDocs.js [hunk 2]
```diff
@@ -0,0 +1,180 @@
...
+categoryName: ReactNative SDK
+title: ${originalFilename}
+slug: ${slug}
+excerpt: None
+category: 60b289fa3ada0c007f41d5a9
```
Comment (ppolus-miquido) [inline scripts/convertDocs.js:59]: Whats the source of this id?

**Review by ppolus-miquido — APPROVED** [review]

File: scripts/convertDocs.js (hunk 1 above)
Comment (arkadiuszkrykM) [reply scripts/convertDocs.js:54]: This always stays at 1.0

File: scripts/convertDocs.js (hunk 2 above)
Comment (arkadiuszkrykM) [reply scripts/convertDocs.js:59]: This should stay the same, for now we don't need to do anything with it.

**Pull request merged**